app/controllers/access_requests_controller.rb

```diff
@@ -1,5 +1,20 @@
 class AccessRequestsController < ApplicationController
   def index
     @access_requests = AccessRequest.unapproved.order(request_date_utc: :asc)
+
+    @access_request_id = params[:access_request_id]
+    @api_result = params[:api_result]
+  end
+
+  def approve
```

Comment:
This method should be with a bang at the end as it’s state-modifying.

Reply:
Oh, but it's part of a controller, isn't that a bit weird?

Reply:
I've found some prior art on other rails apps and it seems ok!